### PR TITLE
HBASE-23646 Resolved remaining Checkstyle violations in tests of hbase-rest

### PR DESCRIPTION
**Pull request by HorizonNet**
Backport of #990 to branch-1

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   7m 16s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 5 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   7m 15s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   0m 31s |  branch-1 passed with JDK v1.8.0_232  |
| +1 :green_heart: |  compile  |   0m 39s |  branch-1 passed with JDK v1.7.0_242  |
| +1 :green_heart: |  checkstyle  |   5m  6s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   2m 53s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 33s |  branch-1 passed with JDK v1.8.0_232  |
| +1 :green_heart: |  javadoc  |   0m 38s |  branch-1 passed with JDK v1.7.0_242  |
| +0 :ok: |  spotbugs  |   1m  2s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +0 :ok: |  findbugs  |   0m 23s |  branch/hbase-checkstyle no findbugs output file (findbugsXml.xml)  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m  0s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 29s |  the patch passed with JDK v1.8.0_232  |
| +1 :green_heart: |  javac  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 39s |  the patch passed with JDK v1.7.0_242  |
| +1 :green_heart: |  javac  |   0m 39s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   4m 56s |  root: The patch generated 0 new + 4 unchanged - 188 fixed = 4 total (was 192)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   2m 45s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   4m 59s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 30s |  the patch passed with JDK v1.8.0_232  |
| +1 :green_heart: |  javadoc  |   0m 39s |  the patch passed with JDK v1.7.0_242  |
| +0 :ok: |  findbugs  |   0m 13s |  hbase-checkstyle has no data from findbugs  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 13s |  hbase-checkstyle in the patch passed.  |
| +1 :green_heart: |  unit  |  13m 42s |  hbase-rest in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 25s |  The patch does not generate ASF License warnings.  |
|  |   |  64m 25s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1025 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux b9c6ae849a3a 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1025/out/precommit/personality/provided.sh |
| git revision | branch-1 / 248d80b |
| Default Java | 1.7.0_242 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_232 /usr/lib/jvm/zulu-7-amd64:1.7.0_242 |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/1/artifact/out/xml.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/1/testReport/ |
| Max. process+thread count | 2561 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/1/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-rest/src/test/java/org/apache/hadoop/hbase/rest/TestScannersWithFilters.java [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/*
- *
+/**
```
Comment (busbey) [inline hbase-rest/src/test/java/org/apache/hadoop/hbase/rest/TestScannersWithFilters.java]: this is a file header and not a javadoc, shouldn't we be using `/*` and not `/**`?

Comment (HorizonNet) [reply hbase-rest/src/test/java/org/apache/hadoop/hbase/rest/TestScannersWithFilters.java]: Fixed

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 38s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 5 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for branch  |
| -1 :x: |  mvninstall  |   0m  8s |  root in branch-1 failed.  |
| -1 :x: |  compile  |   0m  6s |  hbase-checkstyle in branch-1 failed with JDK v1.8.0_232.  |
| -1 :x: |  compile  |   0m  6s |  hbase-rest in branch-1 failed with JDK v1.8.0_232.  |
| -1 :x: |  compile  |   0m  7s |  hbase-checkstyle in branch-1 failed with JDK v1.7.0_242.  |
| -1 :x: |  compile  |   0m  7s |  hbase-rest in branch-1 failed with JDK v1.7.0_242.  |
| -1 :x: |  checkstyle  |   0m  6s |  The patch fails to run checkstyle in root  |
| -1 :x: |  shadedjars  |   0m  6s |  branch has 11 errors when building our shaded downstream artifacts.  |
| -1 :x: |  javadoc  |   0m  6s |  hbase-checkstyle in branch-1 failed with JDK v1.8.0_232.  |
| -1 :x: |  javadoc  |   0m  6s |  hbase-rest in branch-1 failed with JDK v1.8.0_232.  |
| -1 :x: |  javadoc  |   0m  7s |  hbase-checkstyle in branch-1 failed with JDK v1.7.0_242.  |
| -1 :x: |  javadoc  |   0m  6s |  hbase-rest in branch-1 failed with JDK v1.7.0_242.  |
| +0 :ok: |  spotbugs  |   0m 47s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| -1 :x: |  findbugs  |   0m  6s |  hbase-checkstyle in branch-1 failed.  |
| -1 :x: |  findbugs  |   0m  8s |  hbase-rest in branch-1 failed.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m  8s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m  5s |  hbase-checkstyle in the patch failed with JDK v1.8.0_232.  |
| -1 :x: |  compile  |   0m  6s |  hbase-rest in the patch failed with JDK v1.8.0_232.  |
| -1 :x: |  javac  |   0m  5s |  hbase-checkstyle in the patch failed with JDK v1.8.0_232.  |
| -1 :x: |  javac  |   0m  6s |  hbase-rest in the patch failed with JDK v1.8.0_232.  |
| -1 :x: |  compile  |   0m  7s |  hbase-checkstyle in the patch failed with JDK v1.7.0_242.  |
| -1 :x: |  compile  |   0m  6s |  hbase-rest in the patch failed with JDK v1.7.0_242.  |
| -1 :x: |  javac  |   0m  7s |  hbase-checkstyle in the patch failed with JDK v1.7.0_242.  |
| -1 :x: |  javac  |   0m  6s |  hbase-rest in the patch failed with JDK v1.7.0_242.  |
| -1 :x: |  checkstyle  |   0m  5s |  The patch fails to run checkstyle in root  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| -1 :x: |  shadedjars  |   0m  6s |  patch has 11 errors when building our shaded downstream artifacts.  |
| -1 :x: |  hadoopcheck  |   0m  5s |  The patch causes 11 errors with Hadoop v2.8.5.  |
| -1 :x: |  hadoopcheck  |   0m 11s |  The patch causes 11 errors with Hadoop v2.9.2.  |
| -1 :x: |  javadoc  |   0m  6s |  hbase-checkstyle in the patch failed with JDK v1.8.0_232.  |
| -1 :x: |  javadoc  |   0m  6s |  hbase-rest in the patch failed with JDK v1.8.0_232.  |
| -1 :x: |  javadoc  |   0m  8s |  hbase-checkstyle in the patch failed with JDK v1.7.0_242.  |
| -1 :x: |  javadoc  |   0m  6s |  hbase-rest in the patch failed with JDK v1.7.0_242.  |
| -1 :x: |  findbugs  |   0m  6s |  hbase-checkstyle in the patch failed.  |
| -1 :x: |  findbugs  |   0m  8s |  hbase-rest in the patch failed.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m  7s |  hbase-checkstyle in the patch failed.  |
| -1 :x: |  unit  |   0m  7s |  hbase-rest in the patch failed.  |
| +0 :ok: |  asflicense  |   0m 14s |  ASF License check generated no output?  |
|  |   |   5m 35s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1025 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux f4c8a6762c68 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1025/out/precommit/personality/provided.sh |
| git revision | branch-1 / 907184d |
| Default Java | 1.7.0_242 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_232 /usr/lib/jvm/zulu-7-amd64:1.7.0_242 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-compile-hbase-checkstyle-jdk1.8.0_232.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-compile-hbase-rest-jdk1.8.0_232.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-compile-hbase-checkstyle-jdk1.7.0_242.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-compile-hbase-rest-jdk1.7.0_242.txt |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out//home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1025/out/maven-branch-checkstyle-root.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-shadedjars.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-javadoc-hbase-checkstyle-jdk1.8.0_232.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-javadoc-hbase-rest-jdk1.8.0_232.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-javadoc-hbase-checkstyle-jdk1.7.0_242.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-javadoc-hbase-rest-jdk1.7.0_242.txt |
| findbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-findbugs-hbase-checkstyle.txt |
| findbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/branch-findbugs-hbase-rest.txt |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-checkstyle-jdk1.8.0_232.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-rest-jdk1.8.0_232.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-checkstyle-jdk1.8.0_232.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-rest-jdk1.8.0_232.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-checkstyle-jdk1.7.0_242.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-rest-jdk1.7.0_242.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-checkstyle-jdk1.7.0_242.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-compile-hbase-rest-jdk1.7.0_242.txt |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out//home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1025/out/maven-patch-checkstyle-root.txt |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/xml.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-shadedjars.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-javac-2.8.5.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-javac-2.9.2.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-javadoc-hbase-checkstyle-jdk1.8.0_232.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-javadoc-hbase-rest-jdk1.8.0_232.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-javadoc-hbase-checkstyle-jdk1.7.0_242.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-javadoc-hbase-rest-jdk1.7.0_242.txt |
| findbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-findbugs-hbase-checkstyle.txt |
| findbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-findbugs-hbase-rest.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-unit-hbase-checkstyle.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/artifact/out/patch-unit-hbase-rest.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/testReport/ |
| Max. process+thread count | 46 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/2/console |
| versions | git=1.9.1 maven=3.0.5 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 37s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 5 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 29s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   7m 31s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   0m 30s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  compile  |   0m 38s |  branch-1 passed with JDK v1.7.0_252  |
| +1 :green_heart: |  checkstyle  |   5m  5s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   2m 57s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 37s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 38s |  branch-1 passed with JDK v1.7.0_252  |
| +0 :ok: |  spotbugs  |   1m  3s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +0 :ok: |  findbugs  |   0m 25s |  branch/hbase-checkstyle no findbugs output file (findbugsXml.xml)  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 52s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 32s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javac  |   0m 32s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 38s |  the patch passed with JDK v1.7.0_252  |
| +1 :green_heart: |  javac  |   0m 38s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   4m 54s |  root: The patch generated 0 new + 4 unchanged - 188 fixed = 4 total (was 192)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   2m 51s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   4m 50s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 29s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 37s |  the patch passed with JDK v1.7.0_252  |
| +0 :ok: |  findbugs  |   0m 13s |  hbase-checkstyle has no data from findbugs  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 12s |  hbase-checkstyle in the patch passed.  |
| +1 :green_heart: |  unit  |  14m  5s |  hbase-rest in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 24s |  The patch does not generate ASF License warnings.  |
|  |   |  58m 24s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1025 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux 52901538802f 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1025/out/precommit/personality/provided.sh |
| git revision | branch-1 / 99a328f |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/3/artifact/out/xml.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/3/testReport/ |
| Max. process+thread count | 2033 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/3/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by virajjasani** [comment]
+1

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   7m 13s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 5 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 27s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   7m 25s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   0m 32s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  compile  |   0m 38s |  branch-1 passed with JDK v1.7.0_252  |
| +1 :green_heart: |  checkstyle  |   5m  9s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   2m 55s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 36s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 38s |  branch-1 passed with JDK v1.7.0_252  |
| +0 :ok: |  spotbugs  |   1m  2s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +0 :ok: |  findbugs  |   0m 26s |  branch/hbase-checkstyle no findbugs output file (findbugsXml.xml)  |
| -0 :warning: |  patch  |   1m 17s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 55s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 30s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javac  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 39s |  the patch passed with JDK v1.7.0_252  |
| +1 :green_heart: |  javac  |   0m 39s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   5m  1s |  root: The patch generated 0 new + 4 unchanged - 188 fixed = 4 total (was 192)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   2m 48s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   4m 53s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 30s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 38s |  the patch passed with JDK v1.7.0_252  |
| +0 :ok: |  findbugs  |   0m 14s |  hbase-checkstyle has no data from findbugs  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 12s |  hbase-checkstyle in the patch passed.  |
| +1 :green_heart: |  unit  |  14m 43s |  hbase-rest in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 25s |  The patch does not generate ASF License warnings.  |
|  |   |  65m 40s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1025 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux 85efd5c66d2f 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1025/out/precommit/personality/provided.sh |
| git revision | branch-1 / 99a59cf |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/4/artifact/out/xml.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/4/testReport/ |
| Max. process+thread count | 2290 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1025/4/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**